.github/ISSUE_TEMPLATE/01_bug_report.md

```diff
@@ -0,0 +1,27 @@
+---
+name: 🐜 Bug report
+about: If something isn't working 🔧
+---
+
+### Please describe your environment
+
+At least:
+
+-   Operating System + version (and distro if linux):
+-   Software versions (Java, Terraform, etc)
```

Comment:
maybe we can make it clear to what is used to get there?
- docker container not working on x/y/z?
- if it is a cloud issue, which cloud+tr version etc.?
- if it is a running the java app, which version of x/y/z?
Not sure how to document that, but this would help i think ;)

Reply:
I think that's captured in the steps below

Reply:
maybe, let's test :)